src/pages/GetAssistancePage.tsx

```diff
@@ -37,7 +37,7 @@ function GetAssistancePage({route, account}: GetAssistancePageProps) {
     const menuItems: MenuItemWithLink[] = [
         {
             title: translate('getAssistancePage.chatWithConcierge'),
-            onPress: () => Report.navigateToConciergeChat(),
+            onPress: () => Report.navigateToConciergeChat(true),
```

Comment:
This caused a regression of https://github.com/Expensify/App/issues/41610. We should only dismiss modal when there’s a RHP page.